tests/common/reboot.py

```diff
@@ -254,6 +254,8 @@ def reboot(duthost, localhost, reboot_type='cold', delay=10,
             # get 'wait' and 'timeout' from inventory if they are specified, otherwise use current values
             wait = plt_reboot_ctrl.get('wait', wait)
             timeout = plt_reboot_ctrl.get('timeout', timeout)
+            safe_reboot = plt_reboot_ctrl.get('safe_reboot', safe_reboot)
```

Comment:
Let's keep using safe_reboot with explicitly parameters, the plt_reboot configuration generally is locally, if we set safe_reboot in configuration, it would cause the differentiates being huge on different platforms.
Any reason we have to set safe_reboot in configuration file?

Reply:
Hi @yejianquan, It is used only in places where in the intent is to make sure that the system is fully up before continuing the test. For e.g. in acl, we do a reboot and give a larger wait time for critical process and interfaces to come up and then continue with the tests. The only place that I know of where we don't want to check if critical process is not up and interface is not up is test_reload_config.py because there we want to check almost immediately after reboot, certain conditions. But in all cases we would want to ensure after reboot everything is up. Hence I added that option for the user